src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -144,6 +166,7 @@ ApiGlobe.prototype.addImageryLayer = function addImageryLayer(layer) {
 
 ApiGlobe.prototype.addImageryLayerFromJSON = function addImageryLayerFromJSON(url) {
     return Fetcher.json(url).then((result) => {
+        result.update = updateLayeredMaterialNodeImagery;
```

Comment:
Yes, this change was temporary.
The arbitrary split between elevation|imagery|geometry will be removed too.